controllers/provider-aws/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml

```
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kubernetes
+    role: cloud-controller-manager
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+  - name: metrics
+    port: {{ include "cloud-controller-manager.port" . }}
+    protocol: TCP
+  selector:
+    app: kubernetes
+    role: cloud-controller-manager
+---
+apiVersion: {{ include "deploymentversion" . }}
```

```suggestion
apiVersion: apps/v1
```

(It's not needed anymore)

---

Done.